ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_saas.py

```diff
@@ -301,6 +302,16 @@ def self_test(self):
         raise NotImplementedError
 
 
+@Register(api_type="wca")
+class WCASaaSRoleGenerationPipeline(WCASaaSPipeline, WCABaseRoleGenerationPipeline):
```

Comment:
Do you need this class?

`WCABaseRoleGenerationPipeline` implements it and the default for "SaaS" would be the NOP implementation.

Reply:
Hi Michael, it is a placeholder for (near) future work. Once we got the API from IBM we will implement those Pipelines properly.